h/schemas/admin_group.py

```diff
@@ -0,0 +1,66 @@
+# -*- coding: utf-8 -*-
+
+from __future__ import unicode_literals
+
+import colander
+from deform.widget import SelectWidget, TextInputWidget
+
+from h import i18n
+from h import validators
+from h.models.group import (
+    GROUP_NAME_MIN_LENGTH,
+    GROUP_NAME_MAX_LENGTH,
+    GROUP_DESCRIPTION_MAX_LENGTH
+)
+from h.schemas.base import CSRFSchema
+
+_ = i18n.TranslationString
+
+VALID_GROUP_TYPES = (
+    (u'private', _('Private')),
+    (u'restricted', _('Restricted')),
+    (u'open', _('Open')),
+)
+
+
+class CreateAdminGroupSchema(CSRFSchema):
+
+    group_type = colander.SchemaNode(
+        colander.String(),
+        title=_('Group Type'),
+        widget=SelectWidget(values=((u'', _('Select')),) + VALID_GROUP_TYPES)
```

Comment:
I have some heartburn about the way that the confluence of `colander` and `deform` means putting form-specific presentation stuff in the main schema, but I'm doing my best here...